src/index.test.js

```diff
@@ -1,20 +1,13 @@
 import React from "react";
 import { mount } from "enzyme";
 import Skeleton from "./index";
-import { skeletonClass } from "./skeleton";
-
-const skeletonSelector = "." + String(skeletonClass);
 
 it("should render a skeleton", () => {
   const skeleton = mount(<Skeleton />);
-  const skeletonElements = skeleton.find(skeletonSelector);
-
-  expect(skeletonElements.length).toBe(1);
+  expect(skeleton).toMatchSnapshot();
 });
 
 it("should render the required number of skeletons", () => {
   const skeleton = mount(<Skeleton count={4} />);
-  const skeletonElements = skeleton.find(skeletonSelector);
-
-  expect(skeletonElements.length).toBe(4);
+  expect(skeleton).toMatchSnapshot();
```

Comment:
I'm not a huge fan of snapshots. IMO `expect(skeletonElements.length).toBe(4);` is easier to reason about than parsing through the html of the snapshot file.